ui/app/src/components/AuditManagement/operations.ts

```diff
@@ -66,7 +66,7 @@ export const OperationsMessages = defineMessages({
     defaultMessage: 'Remove Members'
   },
   Add_remote: {
-    id: 'operations.Login_failed',
+    id: 'operations.Add_remote',
```

Comment:
Looks like throughout this whole file, the ids are not inline with the convention for message ids